PyPDF2/_page.py

```diff
@@ -1106,6 +1106,7 @@ def _extract_text(
         self,
         obj: Any,
         pdf: Any,
+        orientations: Tuple[int, ...] = (0, 90, 270, 360),
```

Comment:
Could you please add a it in the docstring with an explanation what it does?

Reply:
updated with proposed design